x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearch.java

```diff
@@ -105,6 +108,8 @@ public class EnterpriseSearch extends Plugin implements ActionPlugin, SystemInde
 
     public static final String CONNECTOR_API_ENDPOINT = "_connector";
 
+    public static final String CONNECTOR_SYNC_JOB_API_ENDPOINT = CONNECTOR_API_ENDPOINT + "/_sync_job";
```

Comment:
Is this the right place for this endpoint name? Should it be in the post command itself? Typically we have higher level definitions here?

Reply:
Fair point, adjusted it with https://github.com/elastic/elasticsearch/pull/102714/commits/91da1f343d1851876fb10a7b475d0a1b09841c09.